kubernetes/cray-tftpd-ipxe/values.yaml

```diff
@@ -53,6 +53,9 @@ cray-service:
   replicaCount: 3
   containers:
     - name: cray-tftp
+      securityContext:
```

Comment:
Shouldn't this be:
`securityContext:
    runAsUser: 0
     runAsGroup: 0
     runAsNonRoot: false
     readOnlyRootFilesystem: false
`

Reply:
Can't get formatting to work nicely, but you get the point with the added values...

Reply:
I can make this change no problem. I just went ahead and applied the hotfix that went into Mercury into this PR.